scanner/src/main/kotlin/scanners/fossid/FossIdConfig.kt

```diff
@@ -39,6 +39,10 @@ import org.ossreviewtoolkit.utils.ort.logger
  * * **"deltaScanLimit":** This setting can be used to limit the number of delta scans to keep for a given repository.
  *   So if another delta scan is created, older delta scans are deleted until this number is reached. If unspecified, no
  *   limit is enforced on the number of delta scans to keep. This property is evaluated only if *deltaScans* is enabled.
+ * * **"detectLicenseDeclaration":** When set, the FossID scan is configured to automatically detect file license
+ *   declarations.
+ * * **"detectCopyrightStatements":** Equivalent to the `detectLicenseDeclaration` configuration, this affects the
```

Comment:
"enables" instead of "affects"? Is it important to mention the other option here? Maybe just use a similar wording:
"When set, the FossID scan is configured to automatically detect copyright statements."